api/src/main/java/io/lumeer/api/model/LinkType.java

```diff
@@ -19,24 +19,94 @@
 
 package io.lumeer.api.model;
 
+import io.lumeer.api.dto.JsonAttribute;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.util.Collections;
 import java.util.List;
 
-public interface LinkType {
+public class LinkType {
+
+   public static final String ID = "_id";
```

Comment:
The same thing here. If you want to refer to this as JSON field mapping, it should be `id` instead.